frontend/src/store/last-will/common/actions.ts

```diff
@@ -17,3 +26,12 @@ export const initLastWillService = async (dispatch: Dispatch<CommonActions>, pay
 		},
 	})
 }
+
+export const addUniqueProgressKeyService = async (dispatch: Dispatch<CommonActions>, payload: AddUniqueProgressKeyPayload) => {
```

Comment:
Diese action is ja nur quasi ein gateway für dispatch. 

Bei manchen Sachen (z.b.) fetches brauchen wir dort mehere functions... Wir könnten bei sachen die das nicht brauchen direkt dispatch durchgeben an die page.tsx und von dort aus callen... Vielleicht wird es dann net so kompliziert mit dem global state. Aber gerne auch so lassen.

Reply:
Check nicht ganz wie du meinst :D

Reply:
wir lassen es so :)